### DISCUSSION TITLE
Bug regarding Quick setting panel

### DISCUSSION THREAD

**Discussion by AliAsgerSaify** [iamrasel/lunar-launcher#85] [General]
As shown in the ss

![Screenshot_2023-08-22-11-36-51-05_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/iamrasel/lunar-launcher/assets/120162464/40c1c072-ef48-48be-9f95-dd717bd07db7)

, when we expand the quick setting panel in notification panel then this type of glitch happens

**Comment by iamrasel** [comment]
I don't think this has anything to do with the launcher. This might be an issue of your device.
